common/src/main/scala/org/apache/celeborn/common/util/PbSerDeUtils.scala

```diff
@@ -309,8 +314,10 @@ object PbSerDeUtils {
       val slavePartitions = slaveLocations.asScala.map(PbSerDeUtils.toPbPartitionLocation).asJava
       val pbWorkerResource = PbWorkerResource.newBuilder()
         .addAllMasterPartitions(masterPartitions)
-        .addAllSlavePartitions(slavePartitions).build()
-      workerInfo.toInfoId() -> pbWorkerResource
```

Comment:
the method `WorkerInfo#toInfoId` has no reference then?

Reply:
Right, it's not used anymore, deleted, thanks.